src/sentry/api/endpoints/group_attachments.py

```diff
@@ -43,8 +43,11 @@ def get(self, request: Request, group) -> Response:
         attachments = EventAttachment.objects.filter(group_id=group.id)
 
         types = request.GET.getlist("types") or ()
+        event_ids = request.GET.get("event_ids")
```

Comment:
do we expect this to allow multiple event ids?
If so, you should use `getlist` here. Otherwise, make the param name singular.